config.php

```diff
@@ -37,15 +37,17 @@
 			'action' => 'comment',
 			'label' => 'question',
 			'comment' => <<<COMMENT
-_This is an automated comment, triggered by adding the label `question`._
-			
-Please note, that the GitHub Issue Tracker is for bug reports and feature requests only.
+Thank you for your question in our issue tracker. In order for this issue tracker to be effective,
+we ask our community to use our online resources specifically designed for support:
```

Comment:
"designed for support" sounds like we're providing paid-degree support. We aren't. It's fully driven by community so we need to state it somehow.